.github/workflows/ci-zk-toolbox-reusable.yml

```diff
@@ -224,100 +230,10 @@ jobs:
         run: |
           ci_run zk_supervisor test upgrade --no-deps --chain era
 
-      - name: Show server_rollup.log logs
-        if: always()
-        run: ci_run cat server_rollup.log || true
 
-      - name: Show server_validium.log logs
+      - name: Upload logs
+        uses: actions/upload-artifact@v4
```

Comment:
This is extremely convenient because reading logs from GH's CI page is terribly slow.